kubernetes/README.md

```diff
@@ -51,20 +51,35 @@ Once complete, create a Kubernetes Secret containing the contents of the `scimse
 kubectl create secret generic scimsession --from-file=scimsession=./scimsession
 ```
 
-## (Optional) Create the Google Workspace Kubernetes secrets
+## For Google Workspace customers only
 
-This section is only relevant to those enrolled in our Google Workspace provisioning beta. Contact your 1Password Sales Representative for more information on this initiative.
+This section is only relevant if Google Workspace is your identity provider. If you are not connecting 1Password to Google Workspace, please proceed to [Deploy to the Kubernetes cluster](https://github.com/1Password/scim-examples/tree/master/kubernetes#deploy-to-the-kubernetes-cluster).
 
-First, you’ll need to edit the file located at `scim-examples/beta/workspace-settings.json`, and fill in the values according to those presented in our Google Workspace documentation.
+### Create a Google service account
+
+You will need to create a Google service account and key as outlined in the 1Password support article [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client). 
```

Comment:
Avoid using [wants and needs](https://support.1password.com/style-guide/#wants-and-needs):

```suggestion
Create a Google service account and key as outlined in the 1Password support article [Connect Google Workspace to 1Password SCIM Bridge](https://support.1password.com/scim-google-workspace/#step-1-create-a-google-service-account-key-and-api-client). 
```